graphql/resolvers.go

```diff
@@ -198,6 +191,58 @@ func (r *mutationResolver) RemoveFavoriteProject(ctx context.Context, identifier
 	}, nil
 }
 
+func (r *mutationResolver) SpawnHost(ctx context.Context, spawnHostInput *SpawnHostInput) (*restModel.APIHost, error) {
```

Comment:
I think we need check and see if the unexpirable host limit has exceeded and return an error if it has

Reply:
This is handled in `NewIntentHost`  It will return an error if they have exceeded their host limit